projects/ngx-prx-styleguide/src/lib/header/header.component.css

```diff
@@ -2,33 +2,31 @@
  * PRX header component
  */
 :host {
+  --header-height: 50px;
+  --logo-width: 72px;
+
   position: fixed;
   top: 0;
   right: 0;
   left: 0;
-  width: 100%;
   max-width: 1200px;
   margin: 0 auto;
-  z-index: 200;
+  z-index: 900;
   border-bottom: 1px solid #e6e6e6;
   background: #1a1a1a;
   padding-left: 20px;
 }
 
-h1 {
-  width: 72px;
-  margin: 0;
-  position: relative;
-  opacity: 0.99;
-  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=99)";
-  filter: alpha(opacity=99);
-  z-index: 900;
+header {
+  display: flex;
+  justify-content: space-between;
 }
+
 h1 a {
-  line-height: 50px;
   display: block;
+  width: var(--logo-width);
```

Comment:
We're still supporting IE11 afaik and CSS custom properties don't work in that browser. I have seen that they weren't all removed. A PR on the new styleguide probably should have happened prior to release . The prior version of the styleguide had a postcss processing step that filled in these vars in global CSS, but it never worked with the component CSS. Does the styleguide now have support for SaSS @pstjean? would that be a better option here?

Reply:
Sorry, I thought we were not supporting IE 11 based on looking at the `browserlist` file (has `not IE 9-11` configured). I can refactor to not use the CSS variables.